src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -81,7 +81,7 @@ impl A320Hydraulic {
             engine_driven_pump_2: EngineDrivenPump::new("YELLOW"),
             blue_electric_pump: ElectricPump::new("BLUE"),
             yellow_electric_pump: ElectricPump::new("YELLOW"),
-            rat: RatPump::new(""),
+            ram_air_turbine: RamAirTurbine::new(""),
```

Comment:
As far as I know the A380 doesn't have multiple RATs. Thus giving a name (or identifier) to a RAT probably isn't necessary. What do you think about removing the argument?

Reply:
Yes probably not needed, A380 only has a HUUUGE one I think.